samples/debug/ppi_trace/src/main.c

```diff
@@ -52,7 +52,7 @@ static void ppi_trace_setup(void)
 	 * driver.
```

Comment:
This comment need to be updated. `README.rst` also still mentions `COMPARE_1` event.

Reply:
@anangl  this is too late to be backported. Please remove the label and let it get merged to master.

Reply:
Removed.